api/src/main/java/com/epam/pipeline/controller/datastorage/DataStorageController.java

```diff
@@ -432,6 +433,23 @@ public Result restoreFileVersion(
         return Result.success();
     }
 
+    @RequestMapping(value = "/datastorage/{id}/list/restore/folder", method = RequestMethod.POST)
```

Comment:
We could use `@PostMapping` annotation here

Reply:
fixed